### ISSUE TITLE
Metadata needed for Knowledge Base entries: Created by, last updated, etc.

### ISSUE THREAD

**Issue by magelm** [NFDI4Microbiota/nfdi4microbiota-knowledge-base#93]
Is it possible to add metadata to the knowledge Base entries (without needing to go to github)?
In my opinion, the necessary metadata should include:

- created by
- date of creation
- last updated

**Comment by mahdi-robbani** [comment]
Hi Maja, I'm not sure I understand the issue.

Github already provides a history of all files, updates to those files and who updated them. You can check the entire history of the repo or go to any individual file on github and click "blame" to see who updated each line. Is there any reason this isn't sufficient?

**Comment by KK-NFDI** [comment]
I agree that a small "last updateed" field would be nice to have somewhere in the header of the KnowledgeBase articles. This makes it possible to see at a glance how up to date the information is. The KnowledgeBase should be all most users ever see. The git repo only for people who want to contribute. So having the info on the KnowledgeBase makes sense.

**Comment by magelm** [comment]
> This makes it possible to see at a glance how up to date the information is. The KnowledgeBase should be all most users ever see. The git repo only for people who want to contribute. 

This was my thought process as well for opening the issue.

**Comment by cpauvert** [comment]
Shouldn't this be possible with the Jekyll template or theme?

**Comment by magelm** [comment]
must-have:

- [X] last updated
- [x] overview of contributors similar to end-of-page overview featured on https://rdmkit.elixir-europe.org/human_pathogen_genomics 

![Image](https://github.com/user-attachments/assets/4a2022e1-69b4-4131-a934-28da0d8842f3)

> is better than featuring only one person.


Wishes:
- [x] whom to get in contact with > done via the above contributors similar to RDMOkit


Irrelevant:
- [ ] date of creation

**Comment by nkurtys** [comment]
check if github pages can access information about who edited and contributed to the pages to show them on the live pages
if not
createa yaml data file collecting profiles of all contributers, which can be referenced in meta data of .md files

**Comment by nkurtys** [comment]
Hey @jvddorpe could you have a look at the contributing.yaml file and fill in the blank spots for everyone or give me all the information in an exec sheet. :)

**Comment by KK-NFDI** [comment]
Closing this as the feature is done. Missing entries in contributors.yaml are in the purview of the individual contributors, as they should decide what and how much to share.
